app/javascript/components/gtl-view.jsx

```diff
@@ -273,7 +273,7 @@ const setPaging = (settings, start, perPage) => ({
 const computePagination = (settings) => ({
   page: settings.current,
   perPage: settings.perpage,
-  perPageOptions: [10, 20, 50, 100, 200, 500, 1000],
+  perPageOptions: [5, 10, 20, 50, 100, 200, 500, 1000],
```

Comment:
As discussed, this was added to match the Setting side, where 5 is already a choice.